dev/ci/ci-elpi.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+source ${ci_dir}/ci-common.sh
+
+Elpi_CI_DIR=${CI_BUILD_DIR}/elpi
+
+git_checkout ${Elpi_CI_BRANCH} ${Elpi_CI_GITURL} ${Elpi_CI_DIR}
+
+( cd ${Elpi_CI_DIR} && make COQBIN=$PWD/bin/ && make install COQBIN=$PWD/bin/ )
```

Comment:
Why are we touching COQBIN here? It's exported in ci-common and is different for gitlab.

Reply:
oh, thanks I did not know